src/tests/ftest/util/dmg_utils.py

```diff
@@ -897,6 +897,46 @@ def system_stop(self, force=False, ranks=None):
                 data[rank] = info[1].strip()
         return data
 
+    def system_leader_query(self):
+        """Query the system MS leader.
+
+        Raises:
+            CommandFailure: if the dmg system stop command fails.
+
+        Returns:
+            dict: a dictionary of host ranks and their unique states.
```

Comment:
Do we need to include "status" from JSON in data dictionary?

Reply:
Yes, I can add the status to the dictionary